src/expr/tconv_seq_proof_generator.cpp

```diff
@@ -135,11 +135,13 @@ theory::TrustNode TConvSeqProofGenerator::mkTrustRewriteSequence(
     }
     else if (pg == nullptr)
     {
-      // maybe the i^th generator can explain it alone
+      // maybe the i^th generator can explain it alone. We may overwrite pg
+      // with this class if another step is encountered in this loop.
```

Comment:
I think it'd be good to say explicitly that unless there is another generator in the loop than this i^th generator *must* be able to explain it alone. This is what was a bit confusing to me.